data/json/items/comestibles/veggy_dishes.json

```diff
@@ -1271,5 +1280,26 @@
     "delete": { "flags": [ "RAW" ] },
     "fun": 1,
     "vitamins": [ [ "vitC", 32 ], [ "calcium", 4 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "groundnut_prepared",
+    "name": { "str_sp": "prepared ground nuts" },
+    "copy-from": "groundnut",
+    "spoils_in": "2 days",
+    "description": "Ground nuts that have been peeled and then boiled, to destroy the harmful protease inhibitors.  Safe to eat, but needs further cooking to become palatable. Can be used much like potato.",
+    "delete": { "flags": [ "RAW" ] },
+    "extend": { "flags": [ "EATEN_HOT", "SMOKABLE" ] },
+    "smoking_result": "dry_veggy",
+    "healthy": 0,
+    "fun": -2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "groundnut_boiled",
+    "name": { "str_sp": "boiled ground nuts" },
+    "copy-from": "groundnut_prepared",
+    "description": "Boiled ground nuts. They look and taste a little like potato, but with a slight nutty taste.",
```

Comment:
```suggestion
    "description": "Boiled ground nuts. They look and taste a little like potatoes, but with a slight nutty taste.",
```

Reply:
Thank you! I've applied all these suggestions!